frontend/packages/kubevirt-plugin/src/utils/storage.ts

```diff
@@ -21,6 +21,7 @@ export const getEmptyInstallStorage = (
   scConfigMap: ConfigMapKind,
   bus = DiskBus.VIRTIO,
   vmName = DUMMY_VM_NAME,
+  size = '20Gi',
```

Comment:
do we have a constant for default '20'? if not can you create it

Reply:
constant created